google/cloud/spanner/options.h

```diff
@@ -361,6 +361,17 @@ struct CommitReturnStatsOption {
   using Type = bool;
 };
 
+/**
+ * Option for `google::cloud::Options` to set a maximum delay for
+ * the committed transaction in order to improve throughput. Value must be
```

Comment:
I would suggest including more of the text from the proto so that the user has more information to decide whether they should look into this.
```
... to set the amount of latency this request is willing to incur in order
to improve throughput. If this field is not set, Spanner assumes requests
are relatively latency sensitive and automatically determines an appropriate
delay time.
```

I might also suggest `s/throughput/overall throughput` as it certainly sounds like this app's throughput will not be improved.

Reply:
I've included more of the test from the proto.

Per `s/throughput/overall throughput`: This will improve the throughput of the setter. It will make the latency worse, But increasing this value will improve your throughput. Maybe I'm misunderstanding?